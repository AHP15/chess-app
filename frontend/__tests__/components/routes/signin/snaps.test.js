import renderer from 'react-test-renderer';
import SignIn from '../../../../src/components/routes/signin';

it('displays the right title', () => {
  const component = renderer.create(<SignIn />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});