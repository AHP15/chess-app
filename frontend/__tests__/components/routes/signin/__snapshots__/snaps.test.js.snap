// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the right title 1`] = `
<div>
  <p
    className="text-3xl font-bold underline"
  >
    SignIn page
  </p>
  <p
    className="counter"
  >
    0
  </p>
  <button
    onClick={[Function]}
  >
    Click me
  </button>
</div>
`;
